packages/js/src/generators/release-version/release-version.ts

```diff
@@ -993,7 +999,8 @@ let pmVersion: string | undefined;
 const localPackageProtocols = [
   'file:', // all package managers
   'workspace:', // not npm
-  'portal:', // modern yarn only
```

Comment:
I had jumped the gun on this being there in a previous PR, because we don't support it at the project graph level right now, so it's already not an applicable setup